### DISCUSSION TITLE
Misleading name of `FlagAndOpenMode`  [node]

### DISCUSSION THREAD

**Discussion by Vampire** [DefinitelyTyped/DefinitelyTyped#64140] [Issues with a @types package] [answered]
In `types/node/fs/promises.d.ts` there is an interface `FlagAndOpenMode`.
It has two properties, `mode` of type `Mode` and `flag` of type `OpenMode`.
So the name parts in `FlagAndOpenMode` both just refer to the second property.
Maybe it should be `ModeAndOpenMode` or `FileModeAndOpenMode` or something similar.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
@Vampire thanks! I think the name and usage could be actually reviewed (for example, if the usage in fs options can be changed). You can comment on this in the origin PR I think for the author (Simon)
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54330

**Reply by Vampire** [reply]
Thanks, I wrote a comment there

**Comment by SimonSchick** [comment, marked as answer]
I'd just file a PR, not strong feelings about renaming it.

**Reply by Vampire** [reply]
Just to clarify, you will open such a PR right? :-)
I wouldn't know which name to choose that you guys like best. :-)

**Reply by Vampire** [reply]
@SimonSchick?
